test/perf.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Create a read-only connection to the test DB and run some queries that
+# should complete in a reasonable amount of time.
+
+set -e
+
+if [ $# -ne 1 ]; then
+  echo "Expect a single connection string argument."
+  exit 1
+fi
+
+
+# This script only works when CWD is 'test'
+rootdir=`dirname $0`
+pushd $rootdir > /dev/null
+
+source common.sh
+trap cleanup EXIT
+
+start_indexer_with_connection_string "$1" "read-only" > /dev/null
+
+wait_for_started
+
+# Disable the test instead of exiting with an error.
+set +e
+if ! IGNORED="$(wait_for_migrated '1')"; then
```

Comment:
At the moment this case is being hit because the mainnet DBs don't have the cumulative rewards changes yet.